float32 signal_level1
float32 signal_level2
float32 signal_level3
